src/applications/pensions/containers/ConfirmationPage.jsx

```diff
@@ -33,77 +27,124 @@ class ConfirmationPage extends React.Component {
 
     return (
       <div>
-        <h3 className="confirmation-page-title">Claim submitted</h3>
-        <p>
-          We process claims in the order we receive them. Please print this page
-          for your records.
-        </p>
-        <p>We may contact you for more information or documents.</p>
+        <h2>Your Veteran's Pension application</h2>
+
+        <va-alert status="success">
```

Comment:
We want to use the v3 version of the VADS components and we can use a utility class to add a margin after the alert.
```suggestion
        <va-alert uswds status="success" class="vads-u-margin-bottom--3">
```